packages/js/src/elementor/replaceVars/elementor-replacevar-plugin.js

```diff
@@ -46,7 +46,12 @@ const getReplacements = ( scope = "" ) => {
 	switch ( scope ) {
 		case "post":
 			return [
+				"authorFirstName",
```

Comment:
this is not the complete list of replace vars - is that a conscious decision?

Reply:
You reviewed a bit too early. @leonidasmi and I are still implementing it 🙂 .